Worker Docker image builds are consistently failing

Worker Docker image builds [are consistently failing](https://github.com/codalab/codalab-worksheets/runs/1923691443) with the following error:

```
Step 1/25 : FROM golang as ecr-login-installation
latest: Pulling from library/golang
0ecb575e629c: Already exists
7467d1831b69: Already exists
feab2c490a3c: Already exists
f15a0f46f8c3: Already exists
1517911a35d7: Pulling fs layer
8e11185b0368: Pulling fs layer
f496c6a12b74: Pulling fs layer
f496c6a12b74: Verifying Checksum
f496c6a12b74: Download complete
1517911a35d7: Verifying Checksum
1517911a35d7: Download complete
8e11185b0368: Verifying Checksum
8e11185b0368: Download complete
1517911a35d7: Pull complete
8e11185b0368: Pull complete
f496c6a12b74: Pull complete
Digest: sha256:f3f90f4d30866c1bdae90012b506bd5e557ce27ccd2510ed30a011c44c1affc8
Status: Downloaded newer image for golang:latest
 ---> 861b1afd1d13
Step 2/25 : ENV GOPATH /root/go
 ---> Running in cb09c566ee86
Removing intermediate container cb09c566ee86
 ---> 0424ce2b60ab
Step 3/25 : RUN go get -u github.com/awslabs/amazon-ecr-credential-helper/ecr-login/cli/docker-credential-ecr-login
 ---> Running in 0aa8a6018331
go: downloading github.com/awslabs/amazon-ecr-credential-helper/ecr-login v0.0.0-20210211190242-b19192b6522b
go: downloading github.com/awslabs/amazon-ecr-credential-helper v0.5.0
go: downloading github.com/docker/docker-credential-helpers v0.6.3
go: downloading github.com/sirupsen/logrus v1.4.2
go: downloading github.com/mitchellh/go-homedir v1.1.0
go: downloading github.com/aws/aws-sdk-go v1.36.0
go: downloading github.com/sirupsen/logrus v1.8.0
go: downloading github.com/aws/aws-sdk-go v1.37.13
go: downloading github.com/pkg/errors v0.9.1
go: downloading github.com/konsorten/go-windows-terminal-sequences v1.0.2
go: downloading github.com/konsorten/go-windows-terminal-sequences v1.0.3
go: downloading golang.org/x/sys v0.0.0-20200930185726-fdedc70b468f
go: downloading golang.org/x/sys v0.0.0-20210217105451-b926d437f341
go: downloading github.com/jmespath/go-jmespath v0.4.0
go: downloading github.com/magefile/mage v1.10.0
go: downloading github.com/magefile/mage v1.11.0
Removing intermediate container 0aa8a6018331
 ---> 7400aff19a04
Step 4/25 : WORKDIR /root/go/src/github.com/awslabs/amazon-ecr-credential-helper
 ---> Running in 5ad4feaf50a3
Removing intermediate container 5ad4feaf50a3
 ---> b4b50cccc714
Step 5/25 : RUN make
 ---> Running in ebd0857d9a39
make: *** No targets specified and no makefile found.  Stop.
The command '/bin/sh -c make' returned a non-zero code: 2
[CodaLab] Building worker image
docker pull ***/worker:master

docker build --cache-from ***/worker:master  -t ***/worker:master -f docker_config/dockerfiles/Dockerfile.worker .
Traceback (most recent call last):
  File "***_service.py", line 918, in <module>
    main()
  File "***_service.py", line 112, in main
    service_manager.execute()
  File "***_service.py", line 652, in execute
    self.build_images()
  File "***_service.py", line 905, in build_images
    self.build_image(image)
  File "***_service.py", line 699, in build_image
    % (cache_args, build_args, docker_image, image)
  File "***_service.py", line 721, in _run_docker_cmd
    raise e
  File "***_service.py", line 716, in _run_docker_cmd
    subprocess.check_call(command_string, shell=True, cwd=BASE_DIR)
  File "/opt/hostedtoolcache/Python/3.6.12/x64/lib/python3.6/subprocess.py", line 311, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'docker build --cache-from ***/worker:master  -t ***/worker:master -f docker_config/dockerfiles/Dockerfile.worker .' returned non-zero exit status 2.
```

--- a/docker_config/dockerfiles/Dockerfile.worker
+++ b/docker_config/dockerfiles/Dockerfile.worker
@@ -1,4 +1,4 @@
-FROM golang as ecr-login-installation
+FROM golang:1.15 as ecr-login-installation
 # Install the Amazon ECR Docker Credential Helper
 # Use Docker multi-stage builds to get rid of intermediate layers related to installation
 

